src/ui/components/Settings.tsx

```diff
@@ -15,7 +15,10 @@ tabs[SettingsTabs.OPTIONS] = (props: SettingsProps) => (
             item: SettingsTabs.TRACKS,
             label: props.data.getTranslation('Quality'),
             info: `${
-              props.data.activeTrack ? props.data.activeTrack.width : ''
+              props.data.activeTrack ?
+              (props.data.activeTrack.width ?
+                  (""+props.data.activeTrack.width+"x"+props.data.activeTrack.height+", "):"")
+              +(props.data.activeTrack.bandwidth/1e6).toPrecision(2)+"M" : ''
```

Comment:
Not a big fan of this line, ES6 template literals would be a cleaner approach imo.